resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing an account ID that does not exist</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>accountId: A1234562</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountClosureReason": "All funds withdrawn",
+          "closureDate": "2017-05-20"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with an invalid 'Accept' header</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>accountId: 1234567890<br><br>Accept: application/vnd.hmrc.1.0</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountClosureReason": "All funds withdrawn",
+          "closureDate": "2017-05-20"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: closeExistingLisaAccount
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountRequest'
+            examples:
+              example-1:
+                value:
+                  accountClosureReason: All funds withdrawn
+                  closureDate: 2017-05-20
+              example-2:
+                value:
+                  accountClosureReason: Cancellation
+                  closureDate: 2017-05-20
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountResponse'
```

Comment:
The `example` facet is missing here so it's defaulting to wrong values as shown in the snippet. Please add, thanks.

<img width="348" alt="Screenshot 2023-02-08 at 11 21 53" src="https://user-images.githubusercontent.com/63714619/217516820-1b014b0b-b21f-4f70-a528-c25bea5659ed.png">

      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountResponse'
              example:
                status: 200
                success: true
                data:
                  message: LISA account closed
                  accountId: '1234567890'